src/saving/SaveManagerGUI.cs

```diff
@@ -103,8 +107,17 @@ public override void _Ready()
         deleteOldConfirmDialog = GetNode<CustomConfirmationDialog>(DeleteOldConfirmDialogPath);
         saveDirectoryWarningDialog = GetNode<CustomConfirmationDialog>(SaveDirectoryWarningDialogPath);
         errorSaveDeletionFailed = GetNode<CustomConfirmationDialog>(SaveDeletionFailedErrorPath);
+        filterOptions = GetNode<OptionButton>(FilterOptionsPath);
 
         saveList.Connect(nameof(SaveList.OnItemsChanged), this, nameof(RefreshSaveCounts));
+
+        filterOptions.AddItem(TranslationServer.Translate("ALL"), 0);
+        filterOptions.AddItem(TranslationServer.Translate("SAVE_MANUAL"), 1);
+        filterOptions.AddItem(TranslationServer.Translate("SAVE_AUTOSAVE"), 2);
+        filterOptions.AddItem(TranslationServer.Translate("SAVE_QUICKSAVE"), 3);
```

Comment:
Instead of using hard coded numbers here, convert the corresponding `SaveType` to an int

Reply:
What about All saves?

Reply:
You can leave the All saves as is - just leave a comment explaining why it is equal to 0. I think that should suffice.

This would also mean you'd have to manually set `SaveType.Manual` to 1, `SaveType.Autosave` to 2 and so on... in the actual enum.